src/ontology/metadata/medgen.yml

```diff
@@ -0,0 +1,11 @@
+id: MEDGEN
```

Comment:
## Add: `metadata/medgen.yml`
I've marked this PR as 'draft' mainly because this file hasn't been generated correctly. I don't remember at the moment, but I think there is a `make` goal go dynamically generate these `metadata/*.yml` files.

At the time I created this, it was bundled with some other related changes (#375 #363) but now I've made it separate and I also no longer see an impending need for this file quite yet, until MedGen is properly integrated into Mondo.

Reply:
Unless medgen is formally folded in as a source (i.e. mentioned in mondo-ingest-odk.yaml) we should have to add stuff about it?

Reply:
I don't remember my exact line of thinking when I made these changes a few weeks ago, but I think it was (i) should make `prefixes.csv` consistent with `mondo.sssom.config.yml`, which has a MedGen prefix. But (ii) I expected MedGen terms to start coming into the pipeline for some reason, and if that's not the case, we don't need a `.yml` for this.

TLDR: I've just deleted the `.yml`